expression/expression.go

```diff
@@ -40,6 +41,32 @@ type Expression interface {
 
 	// ToString converts an expression into a string.
 	ToString() string
+
+	// SetVarName set values for session variable.
+	SetVarName(string)
```

Comment:
Set session variable is only used by a particular exprssion, we don't need to add it to Expression interface.